tests/unit/test_cli.py

```diff
@@ -84,7 +84,9 @@ def test_step():
     for step in tmt.steps.STEPS:
         if step == 'provision':
             continue
-        result = runner.invoke(tmt.cli.main, ['--root', MINI, 'run', step])
+        result = runner.invoke(tmt.cli.main, [
+            '--root', MINI, 'run', '--all', 'provision', '--how=local', step
```

Comment:
Do we rally want to run it locally? I personaly prefer the VM.

Reply:
@pvalena  in travis we cannot run a VM afaict :(

Reply:
@pvalena ack?